packages/flutter/lib/src/widgets/icon_data.dart

```diff
@@ -14,17 +14,24 @@ class IconData {
   ///
   /// Rarely used directly. Instead, consider using one of the predefined icons
   /// like the [Icons] collection.
+  ///
+  /// The [package] argument must be non-null when using a font family that is
+  /// included in a package.
   const IconData(
     this.codePoint, {
-    this.fontFamily,
-  });
+    String fontFamily,
+    this.package,
+  }) : fontFamily = package == null ? fontFamily : 'packages/$package/$fontFamily';
```

Comment:
I think we should either have fontFamily be unaffected by package, and then have a separate getter that returns the combined string, or we should have the package argument not set a field. It's weird having fontFamily and package arguments map to fields but not have their values exactly map.